doc/source/launcher.rst

```diff
@@ -11,3 +11,7 @@ messages by starting ``satpy_launcher.py`` with command-line arguments
 ``-n False -a tcp://<host>:<port>`` where the host and port point to a
 message publisher. Multiple publisher addresses can be given by supplying
 them with additional ``-a`` switches.
+
+The Trollflow2 process can be terminated gracefully by sending SIGTERM
+to the it, eg. by ``kill -SIGTERM <pid>`` where ``<pid>`` is the
+process ID number of Trollflow2.
```

Comment:
One more thing: maybe we should explain that the gracefull termination implies finishing the current running processing?

Reply:
Clarified in https://github.com/pytroll/trollflow2/pull/217/commits/5ecd049b70863022b727d343dea5ca77ae65e135